.travis.yml

```diff
@@ -1,14 +1,13 @@
 language: go
 go:
-  - 1.10.x
+  - 1.11.x
```

Comment:
It should probably run against 1.12 and 1.13.

Reply:
I adjusted it to run against go 1.12 and 1.13.  1.11 is broken and has been by some time by the interface for http.Client changing in some packages, so I have dropped that from the build matrix.  I also generally cleaned up a lot of the travis file since it was doing some things in a weird way.